osbrain/tests/common.py

```diff
@@ -2,17 +2,22 @@
 
 from pytest import mark
 
-skip_windows = mark.skipif(sys.platform == 'win32',
-                           reason='Not supported on windows')
-skip_windows_port_reuse = mark.skipif(sys.platform == 'win32',
-                                      reason='Windows allows port reuse')
-skip_windows_any_port = mark.skipif(sys.platform == 'win32',
-                                    reason='Windows allows binding to well '
-                                    'known ports')
-skip_windows_spawn = mark.skipif(sys.platform == 'win32',
-                                 reason='Windows does not support fork')
-skip_windows_ipc = mark.skipif(sys.platform == 'win32',
-                               reason='Windows does not support IPC')
+skip_windows = mark.skipif(
+    sys.platform == 'win32', reason='Not supported on windows'
+)
+skip_windows_port_reuse = mark.skipif(
+    sys.platform == 'win32', reason='Windows allows port reuse'
+)
+skip_windows_any_port = mark.skipif(
+    sys.platform == 'win32',
+    reason='Windows allows binding to well ' 'known ports',
```

Comment:
You can probably join these two strings

Reply:
Sure, nice catch! :joy:

Fixed in 158568066e6b8279219d91ebe1913c957e407498